qiskit_nature/algorithms/pes_samplers/bopes_sampler.py

```diff
@@ -92,10 +131,12 @@ def __init__(
                     "num_bootstrap must be None or an integer greater than or equal to 2"
                 )
 
-        if isinstance(self._gss.solver, VariationalAlgorithm):  # type: ignore
+        if isinstance(self._gss, GroundStateSolver) and \
```

Comment:
The solver from MinimalEigensolverFactory is not initialized at this point, as the problem is needed in the argument.